prelude/common.js

```diff
@@ -153,7 +154,6 @@ exports.substituteDenominator = function substituteDenominator(f, denominator) {
 };
 
 exports.snapshotify = function snapshotify(file, slash) {
-  assert.strictEqual(file, normalizePath(file));
```

Comment:
Why did you removed this?

Reply:
perfomance hit ?

Reply:
I'll add it back

Reply:
No problem for me if this doens't create possible bugs

Reply:
In fact the snapshotify method is only call here in 6 places

```` javascrript
export default function producer({ backpack, bakes, slash, target, symLinks }) {
  return new Promise((resolve, reject) => {
    if (!Buffer.alloc) {
      throw wasReported(
        'Your node.js does not have Buffer.alloc. Please upgrade!'
      );
    }

    const { prelude } = backpack;
    let { entrypoint, stripes } = backpack;
    entrypoint = snapshotify(entrypoint, slash);
    stripes = stripes.slice();

    const vfs = {};
    for (const stripe of stripes) {
      let { snap } = stripe;
      snap = snapshotify(snap, slash);
      if (!vfs[snap]) vfs[snap] = {};
    }

    const snapshotSymLinks = {};
    for (const [key, value] of Object.entries(symLinks)) {
      const k = snapshotify(key, slash);
      const v = snapshotify(value, slash);
      snapshotSymLinks[k] = v;
```` 

by design, symlink key and value have already been normalized, and snaps as well.
The only one that could potentiall not be normalized here is entrypoint... 

Due to the cost of this assert, I would move it elsewhere to verify that entrypoint is really normalized.

Reply:
Ok agree :+1: